src/data/faq_de.json

```
@@ -376,6 +376,217 @@
+                                    "Wir empfehlen aber weiterhin, sich nach einem positiven Selbsttest eigenverantwortlich so vernünftig zu verhalten, dass keine anderen Personen angesteckt werden. Dann sind weitere Warnungen auch nicht mehr nötig."
+                                ]
+                            },
+                            {
+                                "title": "Einige Tage nach dem letzten positiven Test bin ich beim Selbsttest noch immer positiv. Soll ich nochmal warnen? Warum funktioniert das beim zweiten Mal nicht?",
+                                "anchor": "warn_without_tan_still_positive",
+                                "active": false,
+                                "textblock": [
+                                    "Um einen Missbrauch zu vermeiden oder zu minimieren, dass sinnlose Warnungen getriggert werden, die gar nicht auf realen Risikokontakten beruhen, wurde entscheiden, dass ein Anwender mit der neuen Funktion nur einmal innerhalb einer bestimmten Zeitspanne Andere warnen kann.",
+                                    "Nach Ablauf dieser Zeitspanne ist es dann wieder möglich, auf Basis eines positiven Tests in der App zu warnen.",
+                                    "Wir empfehlen aber weiterhin, sich nach einem positiven Selbsttest eigenverantwortlich so vernünftig zu verhalten, dass keine anderen Personen angesteckt werden."
+                                ]
+                            },
+                            {
+                                "title": "Muss ich ein Foto meines Tests machen, um den Test nachzuweisen?",
+                                "anchor": "warn_without_tan_photo",
+                                "active": false,
+                                "textblock": [
+                                    "Nein, in der Corona-Warn-App ist nicht vorgesehen, Fotos oder andere Dokumente zum Nachweis, dass ein Test wirklich stattgefunden hat, hochzuladen oder anderweitig zu prozessieren. Wir setzen hier bewusst auf die Umsicht und Eigenverantwortung unserer Anwender*innen."
+                                ]
+                            },
+                            {
+                                "title": "Wie oft kann ich warnen?",
+                                "anchor": "warn_without_tan_how_often",
+                                "active": false,
+                                "textblock": [
+                                    "Innerhalb einer bestimmten Periode können Sie einmal ohne TAN warnen. Nach Ablauf dieser Sperrfrist können Sie erneut warnen. Grundsätzlich gibt es nach oben keine Grenzen, wie oft Sie warnen können, solange diese Abstände nicht unterschritten werden. Wir hoffen aber natürlich, dass es keine Notwendigkeit für Sie gibt, in noch kürzeren Abständen immer wieder neu zu warnen."

```

1. Option 1: Just improve content to make it easier to understand. 

```suggestion
                                    "Innerhalb eines bestimmten Zeitraums können Sie einmal ohne TAN warnen. Nach Ablauf der Sperrfrist können Sie erneut ohne TAN warnen. Grundsätzlich gibt es nach oben keine Grenzen, wie oft Sie ohne TAN warnen können, solange die Abstände nicht unterschritten werden. Wir hoffen aber natürlich, dass es keine Notwendigkeit für Sie gibt, in noch kürzeren Abständen immer wieder neu zu warnen."
```

2. Option 2: Remove the last sentence + changes from option 1. While this is certainly a nice sentence, this should be a FAQ where simple answers are given to the questions users have.
```suggestion
                                    "Innerhalb eines bestimmten Zeitraums können Sie einmal ohne TAN warnen. Nach Ablauf der Sperrfrist können Sie erneut ohne TAN warnen. Grundsätzlich gibt es nach oben keine Grenzen, wie oft Sie ohne TAN warnen können, solange die Abstände nicht unterschritten werden."
```